app/classifier/index.cjsx

```diff
@@ -5,6 +5,7 @@ ClassificationSummary = require './classification-summary'
 {Link} = require 'react-router'
 tasks = require './tasks'
 drawingTools = require './drawing-tools'
+{session} = require '../lib/session'
```

Comment:
Could we combine the session id and the timestamp of the last submitted classification in local storage to look for a window period to reset the session key, perhaps say 2-5 mins?